### PR TITLE
Add BBAI-SPI0_1.dts (Fix #24)

### LINKED ISSUE #24: SPI CS configuration for P9.23 on AM572xx, kernel 4.19
I have been working on adapting an industrial application from the BeagleBone Black to the AI and came across the ``v4.19.x-ti-overlays`` branch of the Beagleboard Device Trees repo. Thanks to the great work done here already, I was able to properly configure CAN,UART,etc fairly easily! However, one necessery step I need is to configure P9.23 as a SPI chip select pin. According to the [BeagleBone Cape Header Spreadsheet](https://docs.google.com/spreadsheets/d/1fE-AsDZvJ-bBwzNBj1_sPDrutvEvsmARqFwvbw_HkrE/edit#gid=1518010293) (cell BT58, specifically), this should be an option for pin configuration. However, this doesn't seem to be an option in ``am572x-bone-common-univ.dtsi``, and if I try to add it myself, the resulting overlay will brick my device. I was wondering if someone better acquainted with this project had any ideas about how to add this functionality into the ``v4.19.x-ti-overlays`` branch. The resulting option would appear as something like:

  - 	P9_23_spi_cs_pin: pinmux_P9_23_spi_cs_pin { pinctrl-single,pins = <
		P9_23( PIN_OUTPUT_PULLUP | INPUT_EN | MUX_MODE3)>; };

If someone could even point me in the right direction, I'd be happy to try to implement it myself!

Thanks,

Doug

### PR DESCRIPTION
**Pull request by lorforlinux**
@RobertCNelson @jadonk Please review my code :) Everything has been tested on BBAI.

This PR includes DT overlay for enabling channel@1 of bone_spi0 (only available on BBAI) which fixes #24

### CODE REVIEW

File: src/arm/overlays/BBAI-SPI0_1.dts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (jadonk) [inline src/arm/overlays/BBAI-SPI0_1.dts:1]: Can this not be BONE_SPI0_1?

**Review by jadonk — COMMENTED** [review]
Does this need to be BBAI-specific?

File: src/arm/overlays/BBAI-SPI0_1.dts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ * https://lorforlinux.github.io/GSoC2020_BeagleBoard.org/
+ *
+ * See Cape Interface Spec page for more info on Bone Buses
+ * https://elinux.org/Beagleboard:BeagleBone_cape_interface_spec
+ *
+ * BBAI Virtual cape for /dev/bone/spi/0.1
```
Comment (jadonk) [inline src/arm/overlays/BBAI-SPI0_1.dts:8]: Please add 0.1 to the wiki. It is not currently there.

File: src/arm/overlays/BBAI-SPI0_1.dts (hunk 1 above)
Comment (lorforlinux) [reply src/arm/overlays/BBAI-SPI0_1.dts:1]: It's not available on BBB :(

File: src/arm/overlays/BBAI-SPI0_1.dts (hunk 2 above)
Comment (lorforlinux) [reply src/arm/overlays/BBAI-SPI0_1.dts:8]: I was just waiting for you to merge it updating...

**Pull request merged**